src/Symfony/Component/Config/Tests/Definition/Dumper/YamlReferenceDumperTest.php

```diff
@@ -32,7 +31,7 @@ private function getConfigurationAsString()
 acme_root:
     boolean:              true
     scalar_empty:         ~
-    scalar_null:          ~
+    scalar_null:          null
```

Comment:
Because the [`ExampleConfiguration`](https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Config/Tests/Fixtures/Configuration/ExampleConfiguration.php#L30) explicitly set the default value to `null`.
When setting a default value, [the `YamlReferenceDumper` will replaces the default `~` by the default value set](https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Config/Definition/Dumper/YamlReferenceDumper.php#L96-L112). :)

(This is not related to the changes in this PR though)